src/components/app/content/card/Card.css

```diff
@@ -16,13 +15,43 @@
     justify-content: space-between;
     align-items: center;
     color: white;
+    padding: 1rem;
 }
```

Comment:
I would recommend to use a separate css class for properties that impact sizing and positioning of component layout (width, height, margin, padding, float, etc). The reason is simple - since you've separated card as a standalone component you should have an opportunity to change mentioned options with no changes to the component itself. What if you need to RE-USE it with another positions or sizes? The easiest way to achieve it - to use different classes assigned to a div node wrapping the component. Otherwise if you set it in component itself - inner classes would have a priority over delegated ones.